concept-pwa/src/main/java/mozila/components/concept/pwa/PwaModel.kt

```diff
@@ -0,0 +1,96 @@
+package mozila.components.concept.pwa
+
+import android.graphics.Bitmap
+import org.json.JSONArray
+import org.json.JSONObject
+
+class PwaModel(
+        val short_name: String,
+        val name: String,
+        val display: String,
+        val icon: Icon?,
+        val start_url: String,
+        val theme_color: String,
+        val background_color: String) {
+
+    val pwaIconName: String by lazy {
+        if (short_name.isNotEmpty())
+            return@lazy short_name
+        if (name.isNotEmpty())
+            return@lazy name
+        if (start_url.isNotEmpty())
+            return@lazy start_url
+        ""
+
+    }
+    var iconBitmap: Bitmap? = null
+
+
```

Comment:
\[ktlint\] WARNING: \[no\-consecutive\-blank\-lines\] Needless blank line\(s\) in column 1